data/libs/Passengers.lua

```diff
@@ -179,7 +179,7 @@ function Passengers.GetMaxPassengersForHull(hull, maxMass)
 			end)
 
 			numPassengers = numPassengers + passengers
-			availMass = availMass - cabin.mass
+			availMass = availMass - cabin and cabin.mass or 0
```

Comment:
I'd prefer this and the previous line be wrapped in an `if cabin then ...` block.

Reply:
Fixed in c1e4556d7a1f82be98800599406cef86386ee8e9